packages/demos/stories/Recharts.tsx

```diff
@@ -85,6 +85,33 @@ const DATA = [
 ];
 
 storiesOf('Demos|Recharts', module)
+	.add('lexicon', () => (
```

Comment:
Technically I think all of these demos would be "Lexicon" since they try to follow that specification. If we need to update our line chart, lets just do that instead of making a new chart.